listenbrainz/webserver/static/css/DatePicker.less

```diff
@@ -0,0 +1,342 @@
+	  color: rgb(109, 109, 109);
+	}
+  
+	&__wrapper {
+	  display: flex;
+	  flex-grow: 1;
+	  flex-shrink: 0;
+	  background-color: @pager-bg;
+	  border: 1px solid @pager-border;
+	  border-radius: @pager-border-radius;
+	  min-height: @input-height-base - 2px;// input height minus border*2
+	  padding: 4px 6px;
+	}
+  
+	&__inputGroup {
+	  @digit-width: .54em;
+	  @dot-width: .217em;
+	  // own padding + inputs padding + digits width + dots width
+	  min-width: calc(~"(4px * 3) + " @digit-width * 8 ~" + " @dot-width * 2);
+	  flex-grow: 1;
+	  padding: 0 2px;
+	  box-sizing: content-box;
+  
+	  &__divider {
+		padding: 1px 0;
+		white-space: pre;
+	  }
+  
+	  &__input {
+		min-width: @digit-width;
+		height: 100%;
+		position: relative;
+		padding: 0 1px;
+		border: 0;
+		background: none;
+		font: inherit;
+		box-sizing: content-box;
+		-moz-appearance: textfield;
+  
+		&::-webkit-outer-spin-button,
+		&::-webkit-inner-spin-button {
+		  -webkit-appearance: none;
+		  margin: 0;
+		}
+  
+		&:invalid {
+		  background: fade(red, 10%);
+		}
+  
+		&--hasLeadingZero {
+		  margin-left: -@digit-width;
+		  padding-left: calc(~"1px + " @digit-width);
+		}
+	  }
+	}
+  
+	&__button {
+	  color: @blue;
+	  border: 0;
+	  background: transparent;
+	  padding: 4px 6px;
+  
+	  &:enabled {
+		cursor: pointer;
+  
+		&:hover, &:focus {
+		  .react-date-picker__button__icon {
+			stroke: rgb(0, 120, 215);
+		  }
+		}
+	  }
+  
+	  &:disabled {
+		.react-date-picker__button__icon {
+		  stroke: rgb(109, 109, 109);
+		}
+	  }
+  
+	  svg {
+		display: inherit;
+	  }
+	}
+  
+	&__calendar {
+	  width: 350px;
+	  min-height: 266px;
+	  max-width: 100vw;
+	  position: absolute;
+	  top: 100%;
+	  left: 0;
+	  z-index: 1;
+  
+	  &--closed {
+		display: none;
+	  }
+  
+	  .react-calendar {
+		border-width: thin;
+	  }
+	}
+  }
+
+// .react-clock {
+// 	display: block;
+// 	position: relative;
+  
+// 	&, & *, & *:before, & *:after {
+// 	  -moz-box-sizing: border-box;
+// 	  -webkit-box-sizing: border-box;
+// 	  box-sizing: border-box;
+// 	}
+  
+// 	&__face {
+// 	  position: absolute;
+// 	  top: 0;
+// 	  bottom: 0;
+// 	  left: 0;
+// 	  right: 0;
+  
+// 	  border: 1px solid @dark;
+// 	  border-radius: 50%;
+// 	}
+  
+// 	&__hand {
+// 	  position: absolute;
+// 	  top: 0;
+// 	  bottom: 0;
+// 	  left: 50%;
+// 	  right: 50%;
+  
+// 	  &__body {
+// 		position: absolute;
+// 		background-color: @dark;
+// 		transform: translateX(-50%);
+// 	  }
+// 	}
+  
+// 	&__mark {
+// 	  position: absolute;
+// 	  top: 0;
+// 	  bottom: 0;
+// 	  left: 50%;
+// 	  right: 50%;
+  
+// 	  &__body {
+// 		position: absolute;
+// 		background-color: @dark;
+// 		transform: translateX(-50%);
+// 	  }
+	  
+// 	  &__number {
+// 		position: absolute;
+// 		left: -40px;
+// 		width: 80px;
+// 		text-align: center;
+// 	  }
+// 	}
+  
+// 	&__second-hand {
+// 	  &__body {
+// 		background-color: @orange;
+// 	  }
+// 	}
+//   }
```

Comment:
Are these needed any more? If not let's remove these. We can always view the Git history in case we need them again.

Reply:
Totally forgot about those 😓